fleet/fleet.go

```diff
@@ -0,0 +1,87 @@
+// Package fleet implements a fleet client providing basic operations against a
+// fleet endpoint through fleet's HTTP API. Higher level scheduling and
+// management should be built on top of that.
+package fleet
+
+import (
+	"net"
+	"net/http"
+
+	"github.com/coreos/fleet/client"
+	"github.com/coreos/fleet/etcd"
+	"github.com/coreos/fleet/machine"
+	"github.com/coreos/fleet/registry"
+	"github.com/coreos/fleet/schema"
+	"github.com/coreos/fleet/unit"
+)
+
+// TODO
+type Config struct{}
+
+// TODO
+func DefaultConfig() Config {
+	newConfig := Config{}
+
+	return newConfig
+}
+
+type Status struct {
+	// SystemdStatus represents the status within systemd.
+	SystemdStatus containerstates.State
+
+	// FleetStatus represents the status within fleet.
+	FleetStatus containerstates.State
```

Comment:
The UnitState looks like that. 

```
type UnitState struct {
    Hash string `json:"hash,omitempty"`

    MachineID string `json:"machineID,omitempty"`

    Name string `json:"name,omitempty"`

    SystemdActiveState string `json:"systemdActiveState,omitempty"`

    SystemdLoadState string `json:"systemdLoadState,omitempty"`

    SystemdSubState string `json:"systemdSubState,omitempty"`
}
```

So by name it is al systemd related. Where does your `FleetCurrent` and `FleetDesired` come from?